### ISSUE TITLE
SSMS 18.1 setup blocked

### ISSUE THREAD

**Issue by BachNgX** [MicrosoftDocs/sql-docs#2364]
have installed the SSMS 18.0. Then I accidentally uinstalled it. I tried to install SSMS 18.1, and it appeared that Setup Blocked.

I have tried many ways like delete the file C:\Program Files (x86)\Microsoft SQL Server\140\Tool. And I have deleted all file that related to SSMS. After all, the setup still blocked.

Please help me. Thanks!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5559d56d-e787-2c02-970d-a2825911d81e
* Version Independent ID: 99f7b922-9fca-dc06-f934-d4a87c098764
* Content: [Uninstall an Existing Instance of SQL Server (Setup) - SQL Server](https://docs.microsoft.com/en-us/sql/sql-server/install/uninstall-an-existing-instance-of-sql-server-setup?view=sql-server-2017#feedback)
* Content Source: [docs/sql-server/install/uninstall-an-existing-instance-of-sql-server-setup.md](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/sql-server/install/uninstall-an-existing-instance-of-sql-server-setup.md)
* Product: **sql**
* Technology: **install**
* GitHub Login: @MashaMSFT
* Microsoft Alias: **mathoma**

**Comment by MightyPen** [comment]
@markingmyname  (Issue 2364)  Hi Mark, This is at least the second customer to notify us about an SSMS 18.1 problem.  Can you please look into this.

**Comment by BachNgX** [comment]
Yesterday, I had deleted and uninstalled all component of SQL, then I downloaded SQL server again and SSMS 18.1 still blocked. 
![setup](https://user-images.githubusercontent.com/34470659/60008718-bfe58500-969e-11e9-9999-4078a070b526.png)

**Comment by MashaMSFT** [comment]
Hi @BachNgX , can you please try downloading SSMS from this link? https://go.microsoft.com/fwlink/?linkid=2094583
Do you still see the same error? 

If you do still see the error,  would you mind providing a screenshot indicating the preferred language for your system? You can find that setting with these steps: 
1. Open Settings.
2. Click on Time & Language.
3. Click on Language. 

Thanks!

**Comment by BachNgX** [comment]
I downloaded SSMS from the link and I see the same error 
Here is the screenshot for the system's language
![sl](https://user-images.githubusercontent.com/34470659/60066207-a260fb80-9730-11e9-9b87-cbe729895a95.png)

**Comment by MashaMSFT** [comment]
Hi @BachNgX , interesting, in that case, that installation should work. Is it possible that it's being blocked by some kind of security settings? Would you be able to test on a different machine?

Thanks!

**Comment by BachNgX** [comment]
Hi @MashaMSFT , Can you check the log file for me, please? I don't have another machine. Sorry.

Thanks!
[logfile.txt](https://github.com/MicrosoftDocs/sql-docs/files/3323435/logfile.txt)

**Comment by MashaMSFT** [comment]
Hi @BachNgX , unfortunately, if we're not able to rule out your machine / settings as the culprit, there is little more that I can do from the documentation perspective to help you, I am just a simple technical writer :( The log file just states that the installation was blocked. I tried installing with the same installation file, and it worked for me, which is why I suspect something in your environment preventing the installation. 

If you're unable to test with a different machine, please make sure you're launching set up with admin rights. If that doesn't work, I would recommend either opening a support ticket with Microsoft Support directly, or engaging the community for assistance. You can find more information for that here: https://docs.microsoft.com/en-us/sql/sql-server/sql-server-get-help?view=sql-server-2017 

You can also provide your feedback directly to the product group using Uservoice: https://aka.ms/sqlfeedback. 

My apologies that the SQL docs team was unable to assist you further with this, but please let us know if we can do anything more for you from the documentation perspective?

Thanks and wishing you the best! 
Masha from SQL Docs team

**Comment by Matteo-T** [comment]
Hi folks,
Yes, this is indeed a known issue that I'm investigating (see https://feedback.azure.com/forums/908035/suggestions/37483594/, which, by the way, is the best way to report issues/ideas/etc that are not documentation-specific) 

In the interim, in order to get yourself unblocked, could you do this:
1) Close any SSMS you may be running
2) Run regedit.exe and delete this entire key:
    HKLM\SOFTWARE\WOW6432Node\Microsoft\Microsoft SQL Server Management Studio
3) Retry setup.

Thanks,
-Matteo
p.s. Deleting "C:\Program Files (x86)\Microsoft SQL Server\140\Tool" won't do any good to SSMS 18.x, since it installs in a completely different location. Actually, you may have deleted some important files installed by SSMS 17.x and/or SQL Server 2017... so if you are missing something there, you may want to consider repairing those 2 products.
